src/main/java/se/stalhammar/mail/MailHandler.java

```diff
@@ -0,0 +1,39 @@
+package se.stalhammar.mail;
+
+import org.simplejavamail.MailException;
+import org.simplejavamail.email.Email;
+import org.simplejavamail.mailer.Mailer;
+import org.simplejavamail.mailer.MailerBuilder;
+import org.simplejavamail.mailer.config.TransportStrategy;
+
+/**
+ * Wrapper class for SimpleMail mail handler
+ * 
+ * @author Jon Stalhammar
+ *
+ */
+public class MailHandler {
+	private static final String DEFAULT_HOST = "smtp.gmail.com";
+	private static final int DEFAULT_PORT = 587;
+	private static final String DEFAULT_USERNAME = "yourcommutecompanion@gmail.com";
+	private static final String DEFAULT_PASSWORD = "+++commute95";
```

Comment:
Detta ska vi helst mata in som parametrar när vi startar servern. Kan dock sätta det som en TODO, för att göra det enkelt i början.